src/testFixtures/java/org/opensearch/knn/KNNRestTestCase.java

```diff
@@ -852,7 +853,12 @@ protected Request constructScriptScoreContextSearchRequest(
         builder.endObject();
         builder.endObject();
         builder.endObject();
-        Request request = new Request("POST", "/" + indexName + "/_search");
+        String searchParamsString = "";
+        if (searchParams != null) {
```

Comment:
Shall we use isNotEmpty?

Reply:
It's type of Map<String, Object>, will use Objects.nonNull